js/main.js

```diff
@@ -85,14 +86,22 @@ async function main() {
         RESET_COLLECTION.onclick = (event) => {
           localStorage.clear();
           localStorage.setItem("fullCollection", "true");
-          localStorage.setItem("sort", SORT_DROPDOWN.value)
+          localStorage.setItem("sort", SORT_DROPDOWN.value);
+          localStorage.setItem("page-size", CARDS_PER_PAGE_DROPDOWN.value);
```

Comment:
do these need to be set when clicking the reset button? What happens if they aren't?

Reply:
Functionally, nothing, :D
They're there to remember the dropdown values after the reset, If they aren't there, everything will work but if you had "Sort by Rarity" set ,for example, then if you look at the full collection right away it would be sorted with the default, which is collector number. It's to cover a corner case basically.